ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -69,4 +69,11 @@ message InterfaceFilter {
   // Use this to access contract data in a uniform manner in your API client.
   // Optional
   bool include_interface_view = 2;
+
+  // Whether to include a ``create_arguments_blob`` in the returned
+  // ``CreateEvent``.
+  // Use this to access the complete contract data in your API client
+  // for submitting it as a disclosed contract with future commands.
+  // Optional
+  bool include_create_arguments_blob = 3;
```

Comment:
I would argue that this should be promoted to the very top i.e. the `TransactionFilter` message. Here are the reasons why:
- I find it unlikely that anyone would want to issue a heterogeneous subscription request where some contracts have the blob and some not, makes receiving end handling quite difficult. In such cases user will probably opt for 2 separate subscriptions
- A client working with template subscriptions (rather than interfaces) might want to send the blob to another client that works with interfaces.

Reply:
> A client working with template subscriptions (rather than interfaces) might want to send the blob to another client that works with interfaces.

DisclosedContract accepts either blob or record for create_arguments. You have a record in case of a template.

I would maybe push this boolean to the level of interfaceFilter rather than `InclusiveFilters` as we don't necessarily need to link it to the specific interface.

Reply:
Moved to `InclusiveFilters` as agreed offline.

Reply:
in an off-line conversations we have decided not to follow up on this suggestion